### ISSUE TITLE
Consumption Layer QA

### ISSUE THREAD

**Issue by caseyjohnsonwv** [caseyjohnsonwv/totd-ai#5]
Haven't been able to test consumption layer objects because TMX is down tonight. Need to do this and merge `resource-names` back into `main`.

This raises an interesting conversation about fault-tolerance. I've opted for the TMX and OpenPlanet APIs to avoid dealing with Nadeo's API. Not having data 100% of the time is the price you pay for a much faster development cycle and a more stable API. We rely on both TMX and TMIO, and our AI model will require both to accurately make predictions. We don't want to re-train our model with bad or partial data.

In other words, if TMX or OpenPlanet is down, we don't *want* to do anything, so fault tolerance is not a problem on the data ingestion front.
